airflow/providers/google/cloud/operators/dataproc.py

```diff
@@ -3098,6 +3104,7 @@ def execute(self, context: Context):
                 batch_id=self.batch_id,
                 region=self.region,
                 project_id=self.project_id,
+                tags=self.tags,
```

Comment:
The same here. If we are waiting for the batch, `batch_id` is enough to identify it, so we don't have to use rags here.